debian/watch

```diff
@@ -1,11 +1,5 @@
-# Example watch control file for uscan
-# Rename this file to "watch" and then you can run the "uscan" command
-# to check for upstream updates and more.
-# See uscan(1) for format
-
 # Compulsory line, this is a version 3 file
 version=3
 
-
-opts=filenamemangle=s/.+\/v?(\d\S*)\.tar\.gz/matrix-synapse-$1\.tar\.gz/,uversionmangle=s/-rc/~rc/ \
-  https://github.com/matrix-org/synapse/tags .*/v?(\d\S*)\.tar\.gz debian uupdate
+opts=filenamemangle=s/.+\/v?(\d\S*)\.tar\.gz/matrix-synapse-$1\.tar\.gz/,uversionmangle=s/-rc/~rc/,repacksuffix=+dfsg,dversionmangle=s/\+dfsg$//  \
```

Comment:
Right, I will drop the repacking for this branch and resubmit.

Reply:
It's been a while since the last update. Maybe it would be easier to review if we could split this PR into a few independent change sets.

Reply:
It's already split into commits. I think it'd be best if Eric just merges this keeping his delta where really needed, so that we can merge it easier in both ways in future.

Reply:
* I meant splitting into multiple PR.
* Do you see a way to isolate the deltas? This would reduce the burden of keeping things in sync even further.